access/slack/oauth.go

```diff
@@ -0,0 +1,88 @@
+package slack
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/gravitational/teleport-plugins/access/common/auth/oauth"
+	"github.com/gravitational/teleport-plugins/access/common/auth/state"
+	"github.com/gravitational/trace"
+)
+
+// Authorizer implements oauth2.Authorizer for Slack API.
+type Authorizer struct {
+	client *resty.Client
+
+	clientID     string
+	clientSecret string
+}
+
+func newAuthorizer(client *resty.Client, clientID string, clientSecret string) *Authorizer {
+	return &Authorizer{
+		client:       client,
+		clientID:     clientID,
+		clientSecret: clientSecret,
+	}
+}
+
+// NewAuthorizer returns a new Authorizer
+func NewAuthorizer(clientID string, clientSecret string) *Authorizer {
```

Comment:
What are `clientID` and `clientSecret`? Can you add the description to the godoc?